libraries/AC_SafeRTL/AC_SafeRTL.h

```diff
@@ -0,0 +1,89 @@
+#pragma once
+
+#include <AP_Buffer/AP_Buffer.h>
+#include <DataFlash/DataFlash.h>
+#include <AP_HAL/AP_HAL.h>
+#include <AP_Math/AP_Math.h>
+
+#include <bitset>
+#include <vector>
+
+#define SMALL_FLOAT  0.0000001f
```

Comment:
we shouldn't need the SMALL_FLOAT definition.  We've got helper functions like "is_positive()" and "is_equal()" which can be used instead.